.Documentation/articles/innoactive-creator/default-behaviors.md

```diff
@@ -112,105 +146,174 @@ The `Play TextToSpeech Audio` behavior plays speech audio clips generated from t
+
 
 ### Configuration
 
-- #### Object to disable
+- #### Object
+
+    the `Training Scene Object` to be disabled.
 
-    This field contains the `Training Scene Object` to be disabled.
+### Location of this behavior (for developers):
+
+This behavior is part of the [Basic-Conditions-And-Behaviors](https://github.com/Innoactive/Basic-Conditions-And-Behaviors) component. 
 
 ------
 
 ## Enable Object
 
 ### Description
 
-The `Enable Object` behavior **activates** the target's *GameObject* until it will be disabled again.
+This behavior makes the selected `Object` visible and interactive until it is specifically set back to *"disabled"* in a future step.
+Put into Unity terms, it activates the selected Game Object.
+
+### Application Example
+
+- In some scenarios you want trainees to interact with objects that are not visible at the start of the scene. By enabling them they can be “added” to the training scene when needed.
+
+- Use this behavior as an easy way to add additional visual hints.
+
 
 ### Configuration
 
-- #### Object to enable
+- #### Object
+
+    the `Training Scene Object` to be enabled.
 
-    This field contains the `Training Scene Object` to be enabled.
+### Location of this behavior (for developers):
+    
+This behavior is part of the [Basic-Conditions-And-Behaviors](https://github.com/Innoactive/Basic-Conditions-And-Behaviors) component. 
 
 ------
 
 ## Hightlight Object
 
 ### Description
 
-The `Highlight Object` behavior will highlight the target object in the specified color until the end of the step.
+This behavior visually highlights the selected object until the end of a step.
+
+Select the Highlighted `Object` in the Unity Hierarchy and open the Unity Inspector. Search for the *Interactable Highlighter Script*.
+
+[![Interactable Highlighter Script](../images/default-behaviors/interactable-highlighter-script.png "")](../images/default-behaviors/interactable-highlighter-script.png)
+
+You can define the Color and Material for *On Touch Highlight*, *On Grab Highlight*, and *On Use Highlight*. The object will show the highlight color configured in the Highlight behavior by default, as soon as the object is touched it will change to the color configured in *On Touch Highlight*. The same happens when the object is grabbed or used. It will display the configured color in ‘On Grab Highlight’ or ‘On Use Highlight’. 
+
+### Application Example
+
+- Give trainees feedback of objects in the scene they can interact with in the current step. 
+
+- Give trainees feedback when entering an object using their controllers (*On Touch Highlight*) or when grabbing (*On Grab Highlight*) or using (*On Use Highlight*) an object. 
 
 ### Configuration
 
-- #### Hightlight color
+- #### Color
+
+    Color in which the target object will be highlighted. Colors are defined in the RGBA color channel. By configuring the alpha (A) value, highlights can be translucent.
```

Comment:
This does not have to be the RGBA color channel because we are using Unitys color picker where you can also do HSV, or am I missing something?

Reply:
I added HSV color channel as well